eva/executor/load_csv_executor.py

```diff
@@ -0,0 +1,76 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import numpy as np
+import pandas as pd
+
+from eva.configuration.configuration_manager import ConfigurationManager
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.models.storage.batch import Batch
+from eva.planner.load_data_plan import LoadDataPlan
+from eva.storage.storage_engine import StorageEngine
+from eva.readers.csv_reader import CSVReader
+
+
+class LoadCSVExecutor(AbstractExecutor):
+
+    def __init__(self, node: LoadDataPlan):
+        super().__init__(node)
+        config = ConfigurationManager()
+        self.path_prefix = config.get_value('storage', 'path_prefix')
```

Comment:
I suggest doing the path join in the operator node instead of the executor. The executor can simply execute based on the plan.

Reply:
Hmm in the original code, we were joining the path from executor only right? as below. 

```
        # read the video with opencv
        video_reader = OpenCVReader(
            os.path.join(self.path_prefix, self.node.file_path),
            batch_mem_size=self.node.batch_mem_size)
```

Reply:
Do you want me to change this code in the video_executor also?

Reply:
I suggest doing here https://github.com/georgia-tech-db/eva/blob/c2eb49c522f6ab9086a69c9ac2f6bf3747f64ea7/eva/optimizer/statement_to_opr_convertor.py#L226, though we can discuss.